salt/metalk8s/kubernetes/scheduler/installed.sls

```diff
@@ -4,15 +4,11 @@ include:
   - .kubeconfig
 
 Create kube-scheduler Pod manifest:
-  file.managed:
+  metalk8s.static_pod_managed:
     - name: /etc/kubernetes/manifests/kube-scheduler.yaml
     - source: salt://metalk8s/kubernetes/files/control-plane-manifest.yaml
-    - template: jinja
-    - user: root
-    - group: root
-    - mode: 644
-    - makedirs: True
-    - dir_mode: 750
+    - config_files:
+      - /etc/kubernetes/scheduler.conf
```

Comment:
Shouldn't this one have `/etc/kubernetes/pki/ca.crt` just like `k-c-m`?

Reply:
It does not depend on it, neither does it mount the `/etc/kubernetes/pki` directory, so I guess not? In any case, the certificate data is embedded in the generated kubeconfig, and the `metalk8s_kubeconfig: Create kubeconfig file for scheduler` state retrieves the "latest" certificate data from the mine.

Reply:
Hmh, then why would `k-c-m` require `ca.crt`?

Reply:
I put it there because it mounts `/etc/kubernetes/pki` and has a flag for this cert `--cluster-signing-cert-file=/etc/kubernetes/pki/ca.crt`, but maybe it's not required indeed.